proforma/proforma_generator.py

```diff
@@ -518,93 +518,44 @@ def fill_alignment(ws, series, current_row, alignment):
                            number_format='##%')
     current_row += 1
     ws['A{}'.format(current_row)] = "Year one total site carbon dioxide emissions (lb CO2 equivalent)"
```

Comment:
@Bill-Becker can you update the units in each description from "lb CO2 equivalent" to "tCO2"? and then for the health emissions delete "equivalent"